modules/persistent-storage-csi-drivers-supported.adoc

```diff
@@ -67,14 +77,14 @@ ifndef::openshift-dedicated,openshift-rosa[]
 3.
 
 * Does not support offline snapshots or resize. Volume must be attached to a running pod.
+
+4.
+
+* Azure File cloning does not supports NFS protocol. It supports the default storage class, which uses SMB protocol.
```

Comment:
Hi @lpettyjo 
Sorry for the wrong wording comments [here](https://github.com/openshift/openshift-docs/pull/75937#discussion_r1604346731). Can we plz correct like this? 
It supports the default storage class, which uses SMB protocol => It supports with azurefile-csi storage class which uses SMB protocol. 

FI: Default sc will be managed-csi which already supported cloning feature long back.

Reply:
Fixed.